custom_components/healthchecksio/const.py

```
@@ -1,5 +1,13 @@
 """Constants for blueprint."""
+
+from __future__ import annotations
+
+from datetime import timedelta
+
 from homeassistant.const import Platform
+from homeassistant.const import Platform
```

Ruff (0.11.9)

Redefinition of unused `Platform` from line 7

Remove definition: `Platform`

(F811)